browser/src/control/Control.FormulaBarJSDialog.js

```diff
@@ -50,7 +50,8 @@ L.Control.FormulaBarJSDialog = L.Control.extend({
 							command: '.uno:FunctionDialog'
 						},
 						{
-							id: 'autosummenu:AutoSumMenu',
+							id: 'AutoSumMenu',
```

Comment:
this will break menu, it will not work
after ":" we provide id of the menu which should be used

Reply:
in the past we removed ":" from the id of HTML element, it was only used to identify menu...
I think that was changes and caused regression

Reply:
```
{
	id: 'autosummenu:AutoSumMenu',
	type: 'menubutton',
	command: '.uno:AutoSumMenu'
},
```
previously it used `command` because `id` was not unoCommand
(https://github.com/CollaboraOnline/online/commit/1a2500c8d31f12a8e23c648a0afb208e3d09bb05#diff-1951654bafeb4ed39d89489812ff8da269c905feabbe179ccfd312d1f2d19fb6L2485):

Proper fix should be:
1. if `id` has `:` - use first part before `:` as the id of the HTML element
2. change JSON `id: 'autosummenu:AutoSumMenu'` to `id: 'AutoSumMenu:AutoSumMenu'` so it will receive the same id as before